teradatalabs/centos6-ssh-oj8/Dockerfile

```diff
@@ -102,7 +102,7 @@ RUN sed -i \
     /etc/services-config/ssh/sshd_config
```

Comment:
Please add detail to the commit message explaining that the file in question moved to a new location when we updated the base image.